diff -ur a/genesis-0.4.3/setup.py b/genesis-0.4.3/setup.py
--- a/genesis-0.4.3/setup.py	2010-01-02 14:14:40.000000000 +0100
+++ b/genesis-0.4.3/setup.py	2010-02-02 21:28:52.000000000 +0100
@@ -22,14 +22,14 @@
 class Install(install):
     def run(self):
         install.run(self)
-        print 'Running gtk-update-icon-cache'
-        for ICONTHEMEDIR in ICONTHEMEDIRS:
-            try:
-                subprocess.call(
-                    ['gtk-update-icon-cache', '-f', '-t', ICONTHEMEDIR])
-            except OSError:
-                print 'gtk-update-icon-cache not installed, skipping'
-                break
+#        print 'Running gtk-update-icon-cache'
+#        for ICONTHEMEDIR in ICONTHEMEDIRS:
+#            try:
+#                subprocess.call(
+#                    ['gtk-update-icon-cache', '-f', '-t', ICONTHEMEDIR])
+#            except OSError:
+#                print 'gtk-update-icon-cache not installed, skipping'
+#                break
 
 dist = setup(name='genesis',
     version=VERSION,
